pkg/webhook/proxy_webhook.go

```diff
@@ -53,6 +54,7 @@ func (p *proxySetDefaulter) Default(_ context.Context, obj runtime.Object) error
 
 func (p *proxySetDefaulter) DefaultSpec(spec *v1alpha1.ProxySetSpec) {
 	setDefaultServiceArgs(spec)
+	spec.OperatorVersion = pointer.String(v1alpha1.LatestOpVersion.String())
```

Comment:
defaulting op version will cause rolling-update accidentally

Reply:
only set OperatorVersion when it's empty, or distinguish CREATE/UPDATE and only set on CREATE